### PR TITLE
add missing information to return-ref documentation

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: spec/function.dd [hunk 1]
```diff
@@ -1362,8 +1362,8 @@ ref int gun(return ref int x) {
 }
 ---
 
-        $(P Ref methods marked with the `return` attribute ensure the returned
-        reference will not outlive the respective aggregate instance.
+        $(P Struct non-static methods marked with the `return` attribute ensure the returned
+        reference will not outlive the struct instance.
```
Comment (thewilsonator) [inline spec/function.dd]: Classes?

Comment (jacob-carlborg) [reply spec/function.dd]: Interfaces?

Comment (WalterBright) [reply spec/function.dd]: Classes are part of the `return scope` thing.

Comment (thewilsonator) [reply spec/function.dd]: If its going to confuse us, chances are it'll confuse someone else.

**Pull request merged**